tests/unit-tests/test-class-course.php

```diff
@@ -443,15 +443,21 @@ public function testGetViewResultsLinkCourseResultsPage() {
 	 */
 	public function testCourseArchiveOrderSetOrderByDoesNotAddParametersOutsideOfArchivePage() {
```

Comment:
I think I would be useful to add a test for `is_admin` case if it is possible. What do you think?

Reply:
Thanks for this one as well, I have added a test for is_admin too, it usually returns false but made it return true and covered that case, I'll be grateful if you can take a look